apps/right-to-rent-check/translations/src/en/journey.json

```diff
@@ -1,3 +1,3 @@
 {
-  "header": "My New HOF Service"
+  "header": "Home Office right to rent check"
```

Comment:
I wanted it the way you said but I asked Marion the content lead and she wanted as above